src/commands/runScripts/start-dev.js

```diff
@@ -19,9 +19,26 @@ const ready = {
 };
 let appServerProcess;
 
-const log = message => console.log(chalk.yellow('>>'), message);
+const log = (...msgs) => console.log(chalk.yellow('>>'), ...msgs);
+
+// regex to find all instances of the current working directory as a string
```

Comment:
cool cool, will update. The regex replace is mainly just to make the error output a little shorter by stripping out the `/User/mikem/mup-web/` prefixes on all the printed file paths.